src/prop/cadical.h

```diff
@@ -136,6 +142,11 @@ class CadicalSolver : public CDCLTSatSolver, protected EnvObj
   std::vector<SatLiteral> d_assumptions;
 
   unsigned d_nextVarIdx;
+  /** Whether we are logging proofs */
+  bool d_logProofs;
+  /** The proof file */
+  std::string d_pfFile;
+  /** Whether we are in SAT mode */
```

Comment:
This is not very informative but should explain what "SAT mode" refers to.

Reply:
This flag tells us if the SAT solver returned satisfiable so that we know whether we are allowed to query model values from the solver.